.gitignore

```diff
@@ -0,0 +1,10 @@
+
+.vscode/settings.json
+.vs/slnx.sqlite
```

Comment:
Consider ignoring the entire .vs and .vscode directory.

If you've multiple ignores for a single application, consider adding a comment to group them together.